src/main/features/response/tasks/oweMoneyTask.ts

```diff
@@ -7,6 +7,13 @@ export class OweMoneyTask {
       return false
     }
 
+    if (responseDraft.response.type !== ResponseType.OWE_ALL_PAID_NONE) {
```

Comment:
That does not even change anything. I can still choose an option on response type page, be taken to the response details screen, jump to task list without doing anything and see COMPLETED. That is not CORRECT. Task is completed only when all necessary / mandatory details has been captured.

Please correct the logic and reflect it in the unit test.